imagefit/models.py

```diff
@@ -1,12 +1,14 @@
 from __future__ import division
-from imagefit.conf import ext_to_format, settings
+from imagefit.conf import settings
 from PIL import Image as PilImage
 
 import mimetypes
+
 try:
-    import StringIO
+    from StringIO import StringIO
```

Comment:
You use `BytesIO` further down but it won't be available if `StringIO` does not fail importing.